clio-transfer-model/src/main/scala/org/broadinstitute/clio/transfer/model/ubam/UbamMetadata.scala

```diff
@@ -14,7 +15,7 @@ case class UbamMetadata(
   analysisType: Option[Symbol] = None,
   baitIntervals: Option[Symbol] = None,
   baitSet: Option[Symbol] = None,
-  dataType: Option[Symbol] = None,
+  dataType: Option[DataType] = None,
```

Comment:
If you want a 2-for-1, there's a ticket to add DataType to ubam query inputs, too.